gomake

```diff
@@ -81,6 +81,7 @@ gomake_update() {
 clean() {
     echo_green "Cleaning"
     rm -Rf ${work_path}/${target_name}
+    git clean -fdx
```

Comment:
"Remove untracked files from the working tree"

hum ? why would we want to auto remove untracked files ?